<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>371</width>
    <height>284</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout_3" stretch="0,1">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayoutTransformType" stretch="1,0">
      <item>
       <widget class="QComboBox" name="comboBoxTransformType">
        <item>
         <property name="text">
          <string>gray scale</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>high pass</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>low pass</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>paletted</string>
         </property>
        </item>
       </widget>
      </item>
      <item>
       <widget class="QToolButton" name="toolButton">
        <property name="text">
         <string>...</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <item>
       <widget class="QGroupBox" name="groupBox">
        <property name="title">
         <string>Original</string>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_2" stretch="0,1">
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_2">
           <item>
            <widget class="QLineEdit" name="lineEditPath"/>
           </item>
           <item>
            <widget class="QToolButton" name="toolButtonOpenImage">
             <property name="text">
              <string>...</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <widget class="QLabel" name="labelOriginal">
           <property name="text">
            <string>original image</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QGroupBox" name="groupBox_2">
        <property name="title">
         <string>Modified</string>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout" stretch="0,1">
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout">
           <item>
            <spacer name="horizontalSpacer">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>40</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
           <item>
            <widget class="QPushButton" name="pushButtonSaveAs">
             <property name="text">
              <string>Save as ...</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <widget class="QLabel" name="labelModified">
           <property name="text">
            <string>modified image</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
